### PR TITLE
Register models api. Added last_update field, with update information from source

### PR DESCRIPTION
**Pull request by Anntroy**

### CODE REVIEW

File: data_ocean/models.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ class Register(DataOceanModel):
     source_name = models.CharField(max_length=300)
     url_address = models.URLField(max_length=500)
     api_address = models.URLField(max_length=500, null=True)
+    last_update = models.DateTimeField(auto_now_add=True, null=True)
```
Comment (roman-tiukh) [inline data_ocean/models.py:34]: maybe need dataset id? like source_register_id

File: data_ocean/models.py [hunk 2]
```diff
@@ -31,6 +31,7 @@ class Register(DataOceanModel):
     source_name = models.CharField(max_length=300)
```
Comment (roman-tiukh) [inline data_ocean/models.py:31]: i think "name" must be unique

File: location_register/converter/koatuu.py [hunk 3]
```diff
@@ -7,7 +7,7 @@ class KoatuuConverter(Converter):
 
     #paths for the local souce file
     LOCAL_FILE_NAME = "koatuu.json"
-    API_ADDRESS_FOR_DATASET = Register.objects.get(name="Класифікатор об’єктів адміністративно-територіального устрою України (КОАТУУ)").api_address
+    API_ADDRESS_FOR_DATASET = Register.objects.get(name="Класифікатор об’єктів адміністративно-територіального устрою України").api_address
```
Comment (roman-tiukh) [inline location_register/converter/koatuu.py:10]: get by id its may be better than by name, and why name not unique?

File: data_ocean/models.py (hunk 1 above)
Comment (roman-tiukh) [inline data_ocean/models.py:34]: what is last_update? and why auto_now_add?

Comment (roman-tiukh) [reply data_ocean/models.py:34]: we need to add docstring or help_text or verbose_name or just comment to understand what it is.
maybe rename field to source_last_update?

File: data_ocean/models.py (hunk 2 above)
Comment (Anntroy) [reply data_ocean/models.py:32]: Do you think short_name must be added as well?

File: data_ocean/models.py (hunk 1 above)
Comment (Anntroy) [reply data_ocean/models.py]: Fixed source_register_id as URLField

**Pull request merged**